src/search/pruning_method.cc

```diff
@@ -13,6 +16,31 @@ PruningMethod::PruningMethod()
 void PruningMethod::initialize(const shared_ptr<AbstractTask> &task_) {
     assert(!task);
     task = task_;
+    num_unpruned_successors_generated = 0;
+    num_pruned_successors_generated = 0;
+}
```

Comment:
If we use `LimitedPruning`, we keep these three variables for the wrapper as well as for actual pruning method but we count only in `LimitedPruning`. Looks like this is the downside of the new design. Any ideas how to avoid it (without making limitation an option again)?

Reply:
Good point. I don't have any ideas to avoid this problem.